node/modules/lp2p/pubsub.go

```diff
@@ -377,6 +379,7 @@ func GossipSub(in GossipIn) (service *pubsub.PubSub, err error) {
 		build.BlocksTopic(in.Nn),
 		build.MessagesTopic(in.Nn),
 		build.IndexerIngestTopic(in.Nn),
+		gpbft.NetworkName(in.Nn).PubSubTopic(),
```

Comment:
Is there a way to ensure this is enabled only if `build.F3Enabled` is true ? AFAICT, this will always be enabled right now without any gating.